spec/support/features/vote_helpers.rb

```diff
@@ -0,0 +1,10 @@
+module Features
+  module VoteHelpers
+    def vote_setup (user, post)
+      built_post = user.posts.build({ title: post.title, url: post.url })
+      built_post.save
```

Comment:
instead of building an object and then saving it, you could just combine that into one `create` call, would also avoid an unnecessary variable creation